docs/contributor/pocs/otlp-logs.md

```diff
@@ -14,15 +16,14 @@ We created a Helm values file for the `open-telemetry/opentelemetry-collector` c
 
```

Comment:
change line 13 to `## Setup for Log Record Parsing`

Reply:
Done